### DISCUSSION TITLE
Option/lock to prevent moving files/folders around in FileSystem tree

### DISCUSSION THREAD

**Discussion by Zireael07** [godotengine/godot-proposals#5150] [Editor]
Title.
I just accidentally moved a folder, fortunately I noticed and moved it back. Considering we have a Move To... context menu option, moving is still possible w/o dragging, and this would be a lifesaver for people with twitchy/bad touchpad drivers ;)

**Comment by azur-wolve** [comment]
What about have an option to "lock moving files on filesystem"
So when you want to explicitly move a file, can:

- toggle it if are many files
- press some key combination or RMB and select "unlock" to can move them

Meant to avoid moving files by error, maybe good for collaborative projects with multiple users changing stuff simultaneously or the like.

**Comment by Mickeon** [comment]
Chiming on this old discussion. It could be a button right next to the Split Mode toggle.
![image](https://github.com/godotengine/godot-proposals/assets/66727710/d86f0f9f-0f6a-443e-8cd5-5572f9659eaa)

Although, that may be quite odd for a singular feature, so it could also be a button that opens a menu with... File System-related Editor Settings ("Show hidden files" is the only other one I can think about at the moment)
